lib/polarshift/test_suite.rb

```diff
@@ -108,7 +108,8 @@ def commit!(test_case)
         if self.current_test_record.start_scenario_for! test_case
           return true
         else
-          raise "logic error: starting test case that is not part of current test record"
+          puts "logic error: starting test case that is not part of current test record\n"
```

Comment:
We can add the test case id here, for info and reference in failure analysis

Reply:
done